build/stf-run-ci/tasks/setup_base.yml

```diff
@@ -13,10 +13,23 @@
           name: certified-operators
         - disabled: false
           name: redhat-operators
-        - disabled: "{{ false if __service_telemetry_observability_strategy in ['use_community', 'use_hybrid'] else true }}"
+        - disabled: false
           name: community-operators
 
-- name: Create OperatorGroup
+# Upstream Source + Sub from https://github.com/rhobs/observability-operator/tree/main/hack/olm
+# Moved to using Community Operators Catalog, so no longer require upstream CatalogSource. Eventually move to Red Hat Operators CatalogSource.
+- name: Remove CatalogSource for Red Hat Observability Operator (upstream)
+  k8s:
+    state: absent
```

Comment:
Make sure we do a bit of clean up and remove the CatalogSource from previous installations.

Reply:
Actually I might want to move this play to `pre-clean.yml` to keep everything about cleaning up in one spot.

Reply:
I agree that it would be better in pre-clean

Reply:
Moved to pre-clean, and did another test after a deployment that had created the ObO Upstream CatalogSource to fully validate that the pre-clean removed the CS. It does! :)